sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/implementation/model/AccessToken.java

```diff
@@ -108,7 +108,7 @@ public static AccessToken fromJson(JsonReader jsonReader) throws IOException {
                 if ("access_token".equals(fieldName)) {
                     deserializedAccessToken.accessToken = reader.getString();
                 } else if ("expires_in".equals(fieldName)) {
-                    deserializedAccessToken.expiresIn = reader.getLong();
+                    deserializedAccessToken.expiresIn = Long.parseLong(reader.getString());
```

Comment:
Could you please add unit test for it?
What will `getLong()` return here?
How about update `getLong()` method in azure-json?

Reply:
getLong method is OK, just the "expires_in" type of the token response value is not a long value.